### PR TITLE
udevadm-verify: print a short summary

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

**Review by DaanDeMeyer — APPROVED** [review]

File: src/udev/udevadm-verify.c [hunk 1]
```diff
@@ -120,14 +120,30 @@ static int verify_rules_file(UdevRules *rules, const char *fname) {
...
-                        rv = r;
+                if (r < 0) {
+                        fail_count++;
+                        if (rv >= 0)
+                                rv = r;
+                } else
+                        success_count++;
         }
 
+        printf("\n%s%zu udev rules files are checked.%s\n"
```
Comment (ldv-alt) [inline src/udev/udevadm-verify.c]: I'd suggest changing "files are checked" to "files have been checked".

**Review by ldv-alt — COMMENTED** [review]
Ideally, this is worth a test.

**Comment by yuwata** [comment]
@ldv-alt Tests are added. The change is unexpectedly large. Hence, the green label is dropped. PTAL.

File: test/units/testsuite-17.11.sh [hunk 2]
```diff
@@ -9,49 +9,72 @@ set -o pipefail
...
+    local num_str
+    num_str=$(printf %05d "${test_number}")
 
-        rules="sample-${num_str}.rules"
-        exp="sample-${num_str}.exp"
-        err="sample-${num_str}.err"
+    rules="sample-${num_str}.rules"
+    exp="sample-${num_str}.exp"
+    err="sample-${num_str}.err"
+    exo="sample-${num_str}.out"
```
Comment (ldv-alt) [inline test/units/testsuite-17.11.sh]: Please change to
```
exo="sample-${num_str}.exo"
out="sample-${num_str}.out"
```

File: test/units/testsuite-17.11.sh [hunk 3]
```diff
@@ -9,49 +9,72 @@ set -o pipefail
...
 
 assert_0() {
-        assert_rc 0 udevadm verify "$@"
-        next_test_number
+    local output
+
+    output=$(udevadm verify "$@")
+    if [ -f "${rules}" ]; then
+       diff -u "${workdir}/default_output_1_success" <(echo "$output")
+    fi
```
Comment (ldv-alt) [inline test/units/testsuite-17.11.sh]: Please change to
```
udevadm verify "$@" >"${out}"
if [ -f "${rules}" ]; then
    diff -u "${workdir}/default_output_1_success" "${out}"
fi
```

File: test/units/testsuite-17.11.sh [hunk 4]
```diff
@@ -9,49 +9,72 @@ set -o pipefail
...
-                set +e
-                udevadm verify "$@"
-                assert_eq "$?" 1
-                set -e
-        fi
-        next_test_number
+    local output
+
+    set +e
+    output=$(udevadm verify "$@" 2>"${err}")
```
Comment (ldv-alt) [inline test/units/testsuite-17.11.sh]: Please change to
```
udevadm verify "$@" >"${out}" 2>"${err}"
```

File: test/units/testsuite-17.11.sh [hunk 5]
```diff
@@ -9,49 +9,72 @@ set -o pipefail
...
+    assert_eq "$?" 1
+    set -e
+
+    if [ -f "${exp}" ]; then
+        diff -u "${exp}" "${err}"
+    fi
+
+    if [ -f "${exo}" ]; then
+        diff -u "${exo}" <(echo "$output")
```
Comment (ldv-alt) [inline test/units/testsuite-17.11.sh]: Please change to
```
diff -u "${exo}" "${out}"
```

**Review by ldv-alt — COMMENTED** [review]
In the test, please save the output into a file, not into a variable.

**Comment by yuwata** [comment]
@ldv-alt Thanks. The suggestions are applied. Upgrading the green label.

**Pull request merged**